backend/data/src/main/kotlin/io/tolgee/model/UserAccount.kt

```diff
@@ -53,7 +45,14 @@ data class UserAccount(
   var emailVerification: EmailVerification? = null
 
   @Column(name = "third_party_auth_type")
-  var thirdPartyAuthType: String? = null
+  @Enumerated(EnumType.STRING)
+  var thirdPartyAuthType: ThirdPartyAuthType? = null
```

Comment:
I can't understand why this is incompatible with the db schema. `@Enumerated` was already used in the project and worked fine. 🙃

Reply:
Unfortunately, since we are using an already existing column, we have to make sure it is compatible with the previous values. Since we used previously lowercase values like `github` and `google`, we have to either keep using them or write a database migration to migrate them to the new values.

Due to the way the `@Enumerated` works, the real values saved in DB will be in uppercase, making them incompatible with previous values. To fix this, you can write a custom converter that will use your `code()` method instead of the enum's name.